app/models/color_scheme.rb

```diff
@@ -16,6 +16,78 @@ class ColorScheme < ActiveRecord::Base
       "danger" =>            'e45735',
       "success" =>           '1ca551',
       "love" =>              'fa6c8d'
+    },
```

Comment:
lets add a small comment next to each color scheme thanking the particular author of the color scheme for the contribution.